optimizationProblem.m

```diff
@@ -15,23 +15,29 @@
     
     properties (Access = public)
         targetTensor = 1/3*eye(3); %Isotropic encoding tensor
-        N = 50;
+        N = 77;
         initialGuess = 'random';
         useMaxNorm = true;
         gMax = 80;
         sMax = 100;
-        durationFirstPartRequested = 25;
-        durationSecondPartRequested = 25;
-        durationZeroGradientRequested = 0;
-        eta = 1;
+        durationFirstPartRequested = 28;
+        durationSecondPartRequested = 22;
+        durationZeroGradientRequested = 8;
+        eta = .7;
         enforceSymmetry = false;
         redoIfFailed = true;
         name = 'NOW';
+        x0 = [];
+        doMaxwellComp = 1;
+        MaxwellIndex = 60;
     end
     
     properties (SetAccess = private)
         zeroGradientAtIndex = [];
-        tolIsotropy = 1e-4;
+        tolIsotropy = .5e-2; %before 1e-4
+        tolMaxwell
+        s_vec
```

Comment:
What is s_vec?

Reply:
s_vec is the vector of "signs". I think that this is the only way for the optimizer to get values close to zero when integrating over the gradient squared. For a general implementation this vector should switch signs every time a 180 deg pulse is applied.

Reply:
Renamed `s_vec` to `signs`